frontend/src/components/GddSend/TransactionForm.vue

```diff
@@ -177,6 +177,17 @@ export default {
     },
   },
   computed: {
+    disabled() {
```

Comment:
Can we have a solution using the validator?

Reply:
I have tried it but since the inputs are components. this seems somehow not to go. I would need help.